packages/cli/package.json

```diff
@@ -55,9 +59,11 @@
     "command-exists": "^1.2.9",
     "commander": "^11.1.0",
     "console-table-printer": "^2.11.2",
+    "dockerode": "^4.0.2",
     "e2b": "^0.16.2",
     "inquirer": "^9.2.12",
     "open": "^9.1.0",
+    "ssh2": "^1.15.0",
```

Comment:
Added as a peer of `dockerode`, resolves issue in testing: 

```
Can't resolve './crypto/build/Release/sshcrypto.node'
```

Related: 
* https://github.com/apocas/dockerode/issues/775